build/safetyResultsOverTime.js

```diff
@@ -222,8 +225,15 @@ var safetyResultsOverTime = function (webcharts, d3$1) {
 
         //Filter data and nest data by visit and group.
         this.measure_data = this.raw_data.filter(function (d) {
-            return d[_this.config.measure_col] === measure;
+            return d[_this.config.measure_col] === measure && !(_this.config.y.type === 'log' && +d[_this.config.value_col] <= 0);
```

Comment:
I think I'd rather do this in the control instead of deleting all 0 values on pre-process.